medusa/post_processor.py

```diff
@@ -83,22 +87,10 @@ def __init__(self, file_path, nzb_name=None, process_method=None, is_priority=No
 
         self.is_priority = is_priority
 
-        self.log = ''
-
         self.version = None
 
         self.anidbEpisode = None
 
-    def _log(self, message, level=logger.INFO):
-        """
-        A wrapper for the internal logger which also keeps track of messages and saves them to a string for later.
-
-        :param message: The string to log (unicode)
-        :param level: The log level to use (optional)
-        """
-        logger.log(message, level)
-        self.log += message + '\n'
-
```

Comment:
It just shows the important stuff now and I would even reduce it further. The rest can be found in the logs.